File: go/string-contains-substring.go
package main

import (
    "fmt"
    "strings"
)

func main() {
    str := "Fvnjailhgdi"
    fmt.Println(str)
    ss := "jail"
    fmt.Println(ss)

    fmt.Println(strings.Contains(str,ss))
}

File: c/string-contains-substring.c
#include <stdio.h>
#include <string.h>
#include <stdbool.h>

int main() {
    char s[] = "Fvnjailhgdi";
    printf("%s\n", s);
    char ss[] = "jail";
    printf("%s\n", ss);

    bool contains = strstr(s,ss);

    printf("%d\n", contains);
}
